db/migrate/20190410110605_add_column_quality_to_asset.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddColumnQualityToAsset < ActiveRecord::Migration[5.1]
+  def change
+    add_column :assets, :quality, :integer
```

Comment:
I would call this column file_image_quality to be consistent with rest of the fields